File: src/main/java/Meal.java
public class Meal {
    private String name; private boolean vegitarian;
    private double calories; private double price;

    public Meal(String name, double calories, double price, boolean vegitarian) {
        this.name = name; this.vegitarian = vegitarian;
        this.calories = calories; this.price = price;
    }

    public double getPrice() {
        return price;
    }

    public double getCalories() {
        return calories;
    }

    public String getName() {
        return name;
    }

    public boolean isVegitarian() {
        return vegitarian;
    }
}

File: meal.js
public class meal {

    private String name;
    private int calories;
    private int cost;
    private boolean isVeg;

    public meal(String name, int calories, int cost, boolean isVeg);
    this.name =name;
    this.calories =calories;
    this.cost =cost;
    this.isVeg =isVeg;


    public boolean getVeg() {

        return this.isVeg;
    }


    public String getName() {
        return this.name;
    }

    public int getCalories() {

        return this.calories;
    }

    public int getCost() {

        return this.cost;
    }

}
